### ISSUE TITLE
Export and Import of Dashboard is broken

### ISSUE THREAD

**Issue by timhallinflux** [influxdata/chronograf#5739]
__Steps to reproduce:__
Chronograf 1.8.9.1

1. Create a dashboard which relies upon a template variable referencing a specific (not dynamic) source
2. Export the dashboard
3. Create a new organization within Chronograf.
4. Create a new data source with read-only credentials
5. Import the exported dashboard into the new organization
6. The import process prompts to re-map the data source from the exported dashboard to the data source configured within the new org
7. Open dashboard and attempt to edit the template variable -- an error occurs.

```
TypeError: Cannot read property 'id' of undefined
    at a.get (http://localhost:8086/src.6eaf9180.js:3749:2778)
    at a.value (http://localhost:8086/src.6eaf9180.js:3749:1974)
    at Ba (http://localhost:8086/src.6eaf9180.js:717:65834)
    at ja (http://localhost:8086/src.6eaf9180.js:717:65629)
    at $a (http://localhost:8086/src.6eaf9180.js:717:69008)
    at oo (http://localhost:8086/src.6eaf9180.js:717:91055)
    at uo (http://localhost:8086/src.6eaf9180.js:717:91442)
    at Yo (http://localhost:8086/src.6eaf9180.js:717:98524)
    at $o (http://localhost:8086/src.6eaf9180.js:717:97904)
    at eu (http://localhost:8086/src.6eaf9180.js:717:99279)
```

**Comment by timhallinflux** [comment]
it seems like there is a something strange going on with the cookies if you have one org open in one tab and then open another tab and attempt to switch orgs.  Context across those "sessions" is getting mixed up.

**Comment by sranka** [comment]
I tried to reproduce this with 1.8.5 and 1.8.9.1 .... I can't

- exported file differ only in the chronograf version, I used a dashboard that works with any 1.x influxdb : 
[TemplateVariableTest.1.8.9.1.json](https://github.com/influxdata/chronograf/files/6389283/TemplateVariableTest.1.8.9.1.json.txt)
- 1.8.5 asked to remap also variable source, 1.8.9.1 does not ... this is not an issue though, a dynamic source remains dynamic

**Comment by sranka** [comment]
Thank you @timhallinflux for showing how to reproduce this, it shows up that
1. it happens when a template variable is referencing a specific data source
2. Dashboards do not work properly when using more browser tabs with different organizations, it seems like that the session is broken between the tabs.

I do focus on the first case herein first, a sping off the 2nd into a different issue.

**Comment by sranka** [comment]
The actual issue happens in the import, where 

![image](https://user-images.githubusercontent.com/16321466/116638245-307a1e80-a966-11eb-9608-bbafc990e0df.png)

should contain `Variables that use this data source` like this:
![image](https://user-images.githubusercontent.com/16321466/116638409-8f3f9800-a966-11eb-9688-bbb6dd0bbd59.png)

The import then references a data source that does not exist, which is then causing the UI to fail.

**Comment by sranka** [comment]
@timhallinflux this is fixed since chronograf 1.8.10, sources in variables were not mapped prior to 1.8.10

4. #5646: Repair dashboard import to remap also sources in variables.
